lib/ts/BufferWriterFormat.cc

```diff
@@ -60,7 +60,40 @@ tv_to_positive_decimal(ts::TextView src, ts::TextView *out)
 
 namespace ts
 {
-const ts::BWFSpec ts::BWFSpec::DEFAULT;
+const BWFSpec BWFSpec::DEFAULT;
+
+/*
+  'B' : 0x42
+  'b' : 0x62
+  'd' : 0x74
```

Comment:
If I put those in the table, it will become completely unreadable. Using the type flags for initialization is only feasible if using something like the C99 style or writing a specialized initializer list handler.

Reply:
I am not sure I understand. Are you saying changing the value at `0x64` from `0x00` -> `0x03` would make the table unreadable? But wouldn't the code be incorrect otherwise?

Reply:
I meant changing `0x00` to `TYPE_CHAR | NUMERIC_TYPE_CHAR` inside the table would destroy the element alignment in a way that would make it unreadable.

Reply:
Oh I agree. This comment was meant to mark a correction. A separate comment was suggesting a style change. (Just didn't want to forget about this `d` value after I noticed it.)